### PR TITLE
Update Creating Themes Warning

### PR DESCRIPTION
**Pull request by onedrawingperday**
See https://github.com/gohugoio/hugo/issues/5488#issuecomment-447807350 and https://github.com/gohugoio/hugo/issues/5488#issuecomment-447860467 for the rationale of this PR.

@bep Feel free to reword if you think, this could be improved.

### CODE REVIEW

File: content/en/themes/creating.md [hunk 1]
```diff
@@ -20,7 +20,11 @@ wip: true
 ---
 
 {{% warning "Use Absolute Links" %}}
-If you're creating a theme with plans to share it on the [Hugo Themes website](https://themes.gohugo.io/) please note that your theme's demo will be available in a sub-directory of website and for the theme's assets to load properly you will need to create absolute paths in the templates  by using either the [absURL](/functions/absurl) function or `.Permalink`. Also make sure not to use a forward slash `/` in the beginning of a `PATH`, because Hugo will turn it into a relative URL and the `absURL` function will have no effect.
+If you're creating a theme with plans to share it on the [Hugo Themes website](https://themes.gohugo.io/) please note the following: 
+- If you choose to use relative URLs in your template you need to quote them.
```
Comment (bep) [inline content/en/themes/creating.md]: This is a too broad statement, I suggest:

If you use relative URLs in links, they must be quoted, e.g `<a href="{{ "/blog" | relURL }}">` and ``


Or something like that.

File: content/en/themes/creating.md [hunk 2]
```diff
@@ -20,7 +20,11 @@ wip: true
...
+- In style tags you will need to use absolute URLs, for the linked assets to be served properly.
```
Comment (bep) [inline content/en/themes/creating.md]: URLs in inline style, e.g. `<div style="background: url('{{ "images/background.jpg" | absURL }}')">`

Comment (onedrawingperday) [reply content/en/themes/creating.md]: @bep 
I updated the PR following your feedback

File: content/en/themes/creating.md [hunk 3]
```diff
@@ -21,7 +21,6 @@ wip: true
 
 {{% warning "Use Absolute Links" %}}
 If you're creating a theme with plans to share it on the [Hugo Themes website](https://themes.gohugo.io/) please note the following: 
-- If using relative URLs in links, they must be quoted, e.g `<a href="{{ "/blog" | relURL }}">` and `<img src="{{ "/images/logo.png" | relURL }}">`
 - If using inline styles you will need to use absolute URLs, for the linked assets to be served properly, e.g. `<div style="background: url('{{ "images/background.jpg" | absURL }}')">`
```
Comment (bep) [inline content/en/themes/creating.md]: Note that this isn't in a released Hugo version, yet ...

**Comment by onedrawingperday** [comment]
Ping: Since Hugo 0.53 was released, maybe this PR could be merged so that the Docs display the same advice as the README of the Hugo Themes repo.

**Pull request merged**